pkg/nfd-master/nfd-master.go

```diff
@@ -284,6 +295,39 @@ func (m *nfdMaster) Run() error {
 	}
 }
 
+func (m *nfdMaster) processNodeLabelRequest() bool {
+	nodeName, quit := m.queue.Get()
+	if quit {
+		return false
+	}
+	defer m.queue.Done(nodeName)
+	if err := m.nfdAPIUpdateOneNode(nodeName.(string)); err != nil {
+		if m.queue.NumRequeues(nodeName) < 5 {
+			klog.Infof("retrying labeling request for node %s", nodeName)
+			m.queue.AddRateLimited(nodeName)
+		} else {
+			klog.Errorf("error labeling node %v: %v", nodeName, err)
+		}
+	}
+	m.queue.Forget(nodeName)
+	return true
+}
+
+func (m *nfdMaster) runLabeler() {
```

Comment:
I'd suggest to rename this to not being about labeling but about updating nodes `runNodeUpdater` or smth